guides/common/modules/snip_steps-needs-reboot.adoc

```diff
@@ -5,15 +5,15 @@ ifdef::foreman-deb[]
 ----
 $ ls /run/reboot-required
 ----
-. If the `/run/reboot-required` file exists, reboot the system:
+. Optional: If the `/run/reboot-required` file exists, reboot the system:
 endif::[]
 ifndef::foreman-deb[]
 +
 [options="nowrap"]
 ----
 # dnf needs-restarting --reboothint
 ----
-. If the previous command told you to reboot, then reboot the system:
+. Optional: If the previous command told you to reboot, then reboot the system:
 endif::[]
 +
```

Comment:
Those steps don't sound optional at all. Is this SME approved?

Reply:
@adamlazik1 agreed, I guess the idea is that the step is only applicable if the command tells you to reboot. But I agree it seems unnecessary and confusing. Let me check.

Reply:
I think that "optional" tell that you do not have to perform command even if it is applicable, so I'd keep the previous formulation here.

Reply:
This still needs to be resolved.

Reply:
@adamlazik1 while I wait for SME confirmation, I'll proceed with splitting the guide into connected-disconnected guides in this PR. Is that okay?